<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="DecoderFallbackException.xml" source-language="en-US" target-language="ru-RU">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac52fb2ca2e6abfb5b39c24cb0fef8e99e7f2cb0c11.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">2fb2ca2e6abfb5b39c24cb0fef8e99e7f2cb0c11</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Text.DecoderFallbackException">
          <source>The exception that is thrown when a decoder fallback operation fails.</source>
          <target state="translated">Исключение создается при сбое операции резервирования декодера.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Text.DecoderFallbackException">
          <source>This class cannot be inherited.</source>
          <target state="translated">Этот класс не наследуется.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Text.DecoderFallbackException">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Text.DecoderExceptionFallback&gt;</ph> and <ph id="ph2">&lt;xref:System.Text.DecoderFallbackException&gt;</ph> classes.</source>
          <target state="translated">В следующем примере кода показано <ph id="ph1">&lt;xref:System.Text.DecoderExceptionFallback&gt;</ph> и <ph id="ph2">&lt;xref:System.Text.DecoderFallbackException&gt;</ph> классы.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="T:System.Text.DecoderFallbackException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Text.DecoderFallbackException.#ctor">
          <source>The <ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> property for this exception is set to COR_E_ARGUMENT (0x80070057).</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> COR_E_ARGUMENT (0x80070057) имеет значение свойства для этого исключения.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String)">
          <source>An error message.</source>
          <target state="translated">Сообщение об ошибке.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String)">
          <source>A parameter specifies the error message.</source>
          <target state="translated">Через параметр задается сообщение об ошибке.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Text.DecoderFallbackException.#ctor(System.String)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> property for this exception is set to COR_E_ARGUMENT (0x80070057).</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> COR_E_ARGUMENT (0x80070057) имеет значение свойства для этого исключения.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Exception)">
          <source>An error message.</source>
          <target state="translated">Сообщение об ошибке.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Exception)">
          <source>The exception that caused this exception.</source>
          <target state="translated">Исключение, вызвавшее данное исключение.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Exception)">
          <source>Parameters specify the error message and the inner exception that is the cause of this exception.</source>
          <target state="translated">Параметры указывают сообщение об ошибке и внутреннее исключение, вызвавшее данное исключение.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> property for this exception is set to COR_E_ARGUMENT (0x80070057).</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> COR_E_ARGUMENT (0x80070057) имеет значение свойства для этого исключения.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Byte[],System.Int32)">
          <source>An error message.</source>
          <target state="translated">Сообщение об ошибке.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Byte[],System.Int32)">
          <source>The input byte array.</source>
          <target state="translated">Входной массив байтов.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Byte[],System.Int32)">
          <source>The index position in <bpt id="p1">&lt;c&gt;</bpt>bytesUnknown<ept id="p1">&lt;/c&gt;</ept> of the byte that cannot be decoded.</source>
          <target state="translated">Позиция индекса байта, который не удается декодировать, в <bpt id="p1">&lt;c&gt;</bpt>bytesUnknown<ept id="p1">&lt;/c&gt;</ept>.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Byte[],System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.Text.DecoderFallbackException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Byte[],System.Int32)">
          <source>Parameters specify the error message, the array of bytes being decoded, and the index of the byte that cannot be decoded.</source>
          <target state="translated">Параметры указывают сообщение об ошибке, декодируемый массив байтов и индекс байта, который не удается декодировать.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Text.DecoderFallbackException.#ctor(System.String,System.Byte[],System.Int32)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> property for this exception is set to COR_E_ARGUMENT (0x80070057).</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.HResult%2A?displayProperty=nameWithType&gt;</ph> COR_E_ARGUMENT (0x80070057) имеет значение свойства для этого исключения.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" uid="P:System.Text.DecoderFallbackException.BytesUnknown">
          <source>Gets the input byte sequence that caused the exception.</source>
          <target state="translated">Получает входную последовательность байтов, вызвавшую исключение.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Text.DecoderFallbackException.BytesUnknown">
          <source>The input byte array that cannot be decoded.</source>
          <target state="translated">Входной массив байтов, который не удается декодировать.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Text.DecoderFallbackException.BytesUnknown">
          <source>The application uses the <ph id="ph1">&lt;xref:System.Text.DecoderFallbackException.Index%2A&gt;</ph> property to get the position in the input byte array of the byte that cannot be decoded.</source>
          <target state="translated">Приложение использует <ph id="ph1">&lt;xref:System.Text.DecoderFallbackException.Index%2A&gt;</ph> свойство для получения позиции в массиве байтов входного байта, который не удается декодировать.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="P:System.Text.DecoderFallbackException.Index">
          <source>Gets the index position in the input byte sequence of the byte that caused the exception.</source>
          <target state="translated">Получает позицию байта, вызвавшего исключение, во входной последовательности байтов.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Text.DecoderFallbackException.Index">
          <source>The index position in the input byte array of the byte that cannot be decoded.</source>
          <target state="translated">Позиция байта, который не удается декодировать, во входном массиве байтов.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Text.DecoderFallbackException.Index">
          <source>The index position is zero-based.</source>
          <target state="translated">Отсчет позиции начинается с нуля.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Text.DecoderFallbackException.Index">
          <source>The application uses the <ph id="ph1">&lt;xref:System.Text.DecoderFallbackException.BytesUnknown%2A&gt;</ph> property to retrieve the input byte array that contains the byte that cannot be decoded.</source>
          <target state="translated">Приложение использует <ph id="ph1">&lt;xref:System.Text.DecoderFallbackException.BytesUnknown%2A&gt;</ph> свойство для получения входного массива байтов, содержащий байтов, который не удается декодировать.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>